src/scss/components/c-issue-card.scss

```diff
@@ -0,0 +1,13 @@
+.c-issue-card {
```

Comment:
You don't use this style anywhere. Did you forget to push something else?

Reply:
I created it for "issue" index style, you can ignore it too